examples/others/raudio_standalone.c

```diff
@@ -64,6 +64,9 @@
 #if !defined(_WIN32)
 static int kbhit(void);             // Check if a key has been pressed
 static char getch();                // Get pressed character
+#else
+#define kbhit _kbhit
```

Comment:
I think `_kbhit()` and `_getch()` are Visual Studio compiler (and MSVC library) specific...

Reply:
I've changed it over to use the the visual studio #define